hack/setup/README.md

```diff
@@ -0,0 +1,171 @@
+# Setting up a Fleet
+
+This how-to guide discusses how to create a fleet, specifically:
+
+* how to create a hub cluster; and
+* how to join clusters to hub cluster; and
+* how to add labels to a member cluster
+
+> Note
+>
+> To run these scripts, make sure that you have already installed the following tools in your
+> system:
+> * `kubectl`, the Kubernetes CLI
+> * `helm`, a Kubernetes package manager
+> * `curl`
+> * `jq`
+> * `base64`
+
+## Create a hub cluster from a cluster 
+For your convenience, Fleet provides a script that can automate the process of creating a hub cluster. To use script,
+run the commands bellow:
+```sh
+# Replace the value of HUB_CLUSTER_CONTEXT with the name of the kubeconfig context you use for
+# accessing your hub cluster.
+export HUB_CLUSTER_CONTEXT=<YOUR-HUB-CLUSTER-CONTEXT>
+# Replace the value of HUB_CLUSTER_ADDRESS with the address of your hub cluster API server.

```

Comment:
how do we know the hub api server address before we create the hub cluster?

Reply:
Added a note that says to clusters are a prerequisite before running the script and has the azure link to create clusters.